easybuild/easyconfigs/g/GPAW/GPAW-19.8.1-customize-intel.patch

```diff
@@ -0,0 +1,96 @@
+--- /dev/null	2019-08-01 21:28:47.844000266 +0200
```

Comment:
@schiotz Can you add some comments above this line to clarify what this patch does, and why it is needed?

Also mention the author (or upstream reference).

Don't forget to change the checksum for this patch accordingly...